src/Admin/CitizenProjectAdmin.php

```diff
@@ -372,6 +385,10 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
     protected function configureListFields(ListMapper $listMapper)
     {
         $listMapper
+            ->add('turnkeyProject', null, [
```

Comment:
you can remove the template as it does nothing more than call one getter, you can do it with virtual field:
```
->add('projectType', null, [
  'virtual' => true,
  'label' => 'Type de projet',
])
```